src/main/java/ca/on/oicr/gsi/dimsum/controller/mvc/CommonModelAttributeProvider.java

```diff
@@ -2,18 +2,27 @@
 
 import ca.on.oicr.gsi.dimsum.service.CaseService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ModelAttribute;
 
-@ControllerAdvice(basePackages = {"ca.on.oicr.gsi.dimsum.controller.mvc"})
+@ControllerAdvice(basePackages = { "ca.on.oicr.gsi.dimsum.controller.mvc" })
 public class CommonModelAttributeProvider {
 
   @Autowired
   private CaseService caseService;
 
+  @Value("${bugreport.url}")
+  private String bugReportURL;
```

Comment:
Please rename to `bugReportUrl`. There's not really a clear standard for this; just want to be consistent with other naming that includes acronyms/abbreviations (e.g. `RequisitionQc`, `Sample.id`)

Same in model attribute name